components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/RequestObjectValidatorUtil.java

```diff
@@ -245,4 +294,15 @@ public static boolean isSignatureVerified(SignedJWT signedJWT, Certificate x509C
             return false;
         }
     }
+
+    private static boolean isFapiConformant(String clientId) {
+
+        try {
+            return OAuth2Util.isFapiConformantApp(clientId);
+        } catch (IdentityOAuth2Exception e) {
```

Comment:
Throw the exception

Reply:
fixed with 8094265d4abb76d2dd4435c246b499c4a4eb35cf